scrapy/settings/default_settings.py

```diff
@@ -146,6 +146,7 @@
     's3': 'scrapy.extensions.feedexport.S3FeedStorage',
     'ftp': 'scrapy.extensions.feedexport.FTPFeedStorage',
 }
+FEED_STORAGE_BATCH_SIZE = 0
```

Comment:
I think a better default value for this setting, meaning no batch splitting, would be `None`.